### ISSUE TITLE
Pagination with filters

### ISSUE THREAD

**Issue by gadnis** [reststate/reststate-vuex#288]
I hope this is not an issue but my lack of knowledge how to use this package..

I'm trying to make grid component with laravel, argon (bootsrap-vue) and this package. 

Amazing package by the way 😁  sad that it is not maintained any more (maybe readme should include some alternative packages if there is one?)

So, when using loadPage action, there's only pagination data is sent, why not to include filter data? action loadWhere accepts options and filters, but getters.page is empty when using it... is this normal or it can be tweak here?

**Comment by geoidesic** [comment]
@gadnis OP isn't responding anymore to issues even. Look at the JSONAPI website, they list alternatives there. I haven't found a good one yet.

I haven't used pagination yet so I can't advise you there. However also be aware that this library does not fully support the JSONAPI v1.0 specification.

(I've made some modifications which improve this but haven't published them yet – as I was waiting to hear from the OP if he would allow me to take over the npm registry; but he has not replied after several months.)

**Comment by CodingItWrong** [comment]
@geoidesic that's not correct; I replied here in November: https://github.com/reststate/reststate-vuex/issues/283#issuecomment-731734460

**Comment by CodingItWrong** [comment]
@gadnis I'm not comfortable recommending a specific alternate package due to security risks, but now that you mention it, I can add a link to the readme to the JSON:API web site's list of packages

**Comment by geoidesic** [comment]
@CodingItWrong my apologies I didn't see the notification about your reply.
